### DISCUSSION TITLE
Service mesh topology issues

### DISCUSSION THREAD

**Discussion by SergBw** [apache/skywalking#11883] [Q&A]
I configured all my services in Istio mesh to send data in Skywalking as described in the docs:
- https://skywalking.apache.org/docs/main/v9.7.0/en/setup/envoy/als_setting/
- https://skywalking.apache.org/docs/main/v9.7.0/en/setup/envoy/metrics_service_setting/
At first, it worked fine but after some time I found that the topology tab was empty and in the console was an error:
`Uncaught (in promise) TypeError: p is undefined`
At the same time there are no errors in UI and OAP logs.
Maybe you have any ideas about what can cause such issues?

![img](https://github.com/apache/skywalking/assets/51742621/8d526d44-08ad-4ffe-b2e6-b7e79a88999f)

**Comment by wu-sheng** [comment]
This error usually means backend or database is/are both down. 
You need to debug on your own.
